Running MM with the --debug flag allows mmcli to send AT commands
(--command).

Upstream-Status: Inappropriate [configuration specific]

Signed-off-by: Michal Mazurek <michal@resin.io>

--- a/data/ModemManager.service.in	2014-11-25 18:42:10.000000000 +0100
+++ b/data/ModemManager.service.in	2016-09-20 11:41:35.702860625 +0200
@@ -5,7 +5,7 @@
 [Service]
 Type=dbus
 BusName=org.freedesktop.ModemManager1
-ExecStart=@sbindir@/ModemManager
+ExecStart=@sbindir@/ModemManager --debug
 StandardError=null
 Restart=on-abort
 
